### PR TITLE
Adding PHP SDK to docs

### PR DESCRIPTION
**Pull request by orizohar**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

- Adding PHP to list of SDKs
- Adding interlinking to SDKs page

### CODE REVIEW

File: daprdocs/content/en/developing-applications/sdks/_index.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The Dapr SDKs are the easiest way for you to get Dapr into your application. Cho
 | [Python]({{< ref python >}}) | In Development | ✔ | [gRPC]({{< ref python-grpc.md >}}) | [FastAPI]({{< ref python-fastapi.md >}})<br />[Flask]({{< ref python-flask.md >}}) |
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/sdks/_index.md]: Can we put these "In Development" SDK to v1.0 now? Or are you going to do this in another change? If not I would do them in this PR.

Comment (orizohar) [reply daprdocs/content/en/developing-applications/sdks/_index.md]: So you want it changed from "state" to "version"? are they all v1.0? I believe PHP is not.
Maybe for everything that isn't v1.0 we should say "preview" or "planned"

**Review by msfussell — APPROVED** [review]

File: daprdocs/content/en/developing-applications/sdks/_index.md (hunk 1 above)
Comment (orizohar) [reply daprdocs/content/en/developing-applications/sdks/_index.md]: @AaronCrawfis - I know you created this page - thoughts on what this should be for v1.0?

Comment (msfussell) [reply daprdocs/content/en/developing-applications/sdks/_index.md]: v1.0 only for Go, .NET, Java and Python. The rest are preview.

**Pull request merged**